### ISSUE TITLE
Index patterns can be created/removed from kibana in RO mode

### ISSUE THREAD

**Issue by dpose** [sscarduzio/elasticsearch-readonlyrest-plugin#226]
Hi,

I'm testing this plugin to limit the actions an anonymous user can do from kibana. I've set "kibana_access" to "ro", visualizations and dashboards can't be created nor edited nor removed, PERFECT!, but I've noticed I still can create and remove index patterns.
If I set the kibana access to RO, I'd expect nothing can be modified from kibana.

Is it the expected behaviour? Is there any reason to it works that way? Is it an error with my access rules?

This is my elasticsearch.yml file:
```
readonlyrest:
    enable: true
    logic: parallel #defaults to serial
    response_if_req_forbidden: Forbbiden

    access_control_rules:

    - name: "::KIBANA::"
      auth_key: kibana:kibana
      type: allow

    - name: "::RO KIBANA (anonymous)::"
      type: allow
      kibana_access: ro
      indices: [".kibana", "index1"]

    users:

    - username: kibana
      auth_key: kibana
      groups: ["admin"]

    - username: alice
      auth_key: alice
      groups: ["anonymous"]
```

Meanwhile, I avoid the index patterns can be removed limiting the methods kibana can use (only GET, POST and PUT). That way, index patterns still can be created but at least they can't be removed.
```
    - name: "::RO KIBANA (anonymous)::"
      type: allow
      kibana_access: ro
      methods: GET, POST, PUT
      indices: [".kibana", "index1"]
```

**Comment by sscarduzio** [comment]
Hi David,
Nice find! Do you have a log line of the request that removes the index pattern?

In general, we recommend to stay away as much as possible from making elasticsearch-level decisions using HTTP level rules, as it's just not the right level of abstraction.

Sometimes it's  necessary due to absence of document/field level granularity of the rules at the time being.

**Comment by sscarduzio** [comment]
this was fixed recently in 1.16.0
